src/mock.ts

```diff
@@ -19,8 +23,21 @@ const MOCK_API: PenumbraAPI = {
   getTextOrURI: () => [
     Promise.resolve({ data: 'test', type: 'text', mimetype: 'text/plain' }),
   ],
-  zip: async () => new ReadableStream(),
+  saveZip: () =>
+    (({
+      /** Add PenumbraFiles to zip */
+      write(): void {},
```

Comment:
```suggestion
      write(): void {
         return;
      },
```

or   return null; maybe. it doesnt like that there is an empty block here